src/features/list-view/ListView.js

```diff
@@ -105,11 +106,20 @@ class ListView extends React.PureComponent<Props> {
         return canFillWidth ? getClientDefinedColumnWidth(columnIndex) : (width - firstColumnWidth) / (columnCount - 1);
     };
 
+    renderSortIcon(columnIndex: number) {
+        const { getGridHeaderSort = noop } = this.props;
+
+        const sortDirection = getGridHeaderSort(columnIndex);
+        const isSortAsc = sortDirection === SORT_ORDER_ASCENDING;
+
+        return sortDirection && <IconSortChevron className={classNames({ [`${CSS_PREFIX}-isSortAsc`]: isSortAsc })} />;
+    }
+
     render() {
         const { columnCount, height, rowCount, width } = this.props;
 
         return (
-            <div className="metadata-views-list-view">
+            <div className={`${CSS_PREFIX}`}>
```

Comment:
I originally was also following this pattern.
One thing that was brought to my attention by AK was that if we were to do a global search and find on this class name we wouldn't be able to find this class, as it is being set to a const. Is this something we want?

Reply:
I can see the value of a global search. If that's important, then we can tradeoff DRY for searchability.